src/marqo/index.py

```diff
@@ -108,20 +116,29 @@ def create(config: Config, index_name: str,
             settings_dict: if specified, overwrites all other setting
                 parameters, and is passed directly as the index's
                 index_settings
-            inference_node_type: inference type for the index
-            storage_node_type: storage type for the index
-            inference_node_count: number of inference nodes for the index
-            storage_node_count: number of storage nodes for the index
-            replicas_count: number of replicas for the index
+            inference_node_type (deprecated): inference type for the index
```

Comment:
For each of these deprecated parameter, indicate which parameter replaces it. For example:

```
inference_node_type (deprecated): inference type for the index. This parameter is replaced by inference_type.
```